File: csharp+code+xunit/CarPlant/FourWheelDriveCar.cs
namespace CarPlant {
	public abstract class FourWheelDriveCar : Car
	{
		public override void Accelerate(int kph)
		{
			_frontLeft.rotate(kph);
			_frontRight.rotate(kph);
			_rearLeft.rotate(kph);
			_rearRight.rotate(kph);
		}
	}
}

File: python/four_wheel_drive_car.py
from car import Car

class FourWheelDriveCar(Car):

    def accelerate(self, kph):
        self._frontLeft.rotate(kph)
        self._frontRight.rotate(kph)
        self._rearLeft.rotate(kph)
        self._rearRight.rotate(kph)

